commit 27f8e7b5222b522970ff17010d15a046b6cf232e (refs/changes/19/1119/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-14T12:08:35-08:00 (2 years, 10 months ago)
    
    joyent/node-triton#153 upgrade restify-clients to a recent version with explicit transitive deps

diff --git a/package.json b/package.json
index d75237c..8e85e13 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "node-uuid": "1.4.3",
     "once": "1.3.2",
     "read": "1.0.7",
-    "restify-clients": "1.1.0",
+    "restify-clients": "1.5.0",
     "restify-errors": "3.0.0",
     "rimraf": "2.4.4",
     "semver": "5.1.0",
